### DISCUSSION TITLE
Ecu Tweaker 1.8 not show Dacia Duster Phase 3

### DISCUSSION THREAD

**Discussion by iacques** [cedricp/ddt4all#1043] [General]
Hi,
Ecu Tweaker v 1.8 not show Dacia Duster Phase 3, only Duster and Duster Ph2.
With same ecu.zip, ddt4all and Tweaker 2.1 can show Dacia Duster Phase 3, but in this Tweaker  version there is a error when try to connect to bluetooth. I use Elm327 v1.5 obd2 8f25k80 bluetooth.

Jacques

**Comment by Furtif** [comment]
https://github.com/cedricp/ecutweaker/issues/5
